### PR TITLE
Fix transTypeComposite bug

### PR DESCRIPTION
**Pull request by KornevNikita**
This is a workaround to fix an error caused by DIBuilder which always creates
a node with the DW_TAG_struct_type tag for both class and structure.

### CODE REVIEW

File: test/DebugInfo/TransTypeCompositeCaseClass_.ll [hunk 1]
```diff
@@ -0,0 +1,73 @@
+; Source code:
+; class A {};
+; struct B {};
+
+; int main() {
+;   A a;
+;    B b;
+;    return 0;
```
Comment (mlychkov) [inline test/DebugInfo/TransTypeCompositeCaseClass_.ll:8]:
```suggestion
;   B b;
;   return 0;
```

File: lib/SPIRV/SPIRVToLLVMDbgTran.cpp [hunk 2]
```diff
@@ -256,10 +256,10 @@ SPIRVToLLVMDbgTran::transTypeComposite(const SPIRVExtInst *DebugInst) {
   DICompositeType *CT = nullptr;
   switch (Ops[TagIdx]) {
   case SPIRVDebug::Class:
-    CT = Builder.createClassType(
-        ParentScope, Name, File, LineNo, Size, Align, 0, Flags, DerivedFrom,
-        DINodeArray() /*elements*/, nullptr /*VTableHolder*/,
-        nullptr /*TemplateParams*/, Identifier);
+    CT = Builder.createReplaceableCompositeType(
```
Comment (AlexeySachkov) [inline lib/SPIRV/SPIRVToLLVMDbgTran.cpp:259]: I suggest to leave a `TODO` comment here to revert this change once `DIBuilder` is fixed

**Review by AlexeySachkov — APPROVED** [review]
I also suggest to either file a bug in LLVM upstream about this, or propose a patch with fix

**Review by MrSidims — APPROVED** [review]

**Review by AlexeySachkov — APPROVED** [review]

**Review by AlexeySotkin — APPROVED** [review]
